AwesomeTSProject/src/containers/AddCrypto/Index.tsx

```diff
@@ -1,17 +1,78 @@
-import {Modal, View, Pressable, Text} from 'react-native';
+import {Modal, View, Pressable, Alert} from 'react-native';
 import React, {useState} from 'react';
 import {
   CryptoInput,
   AddText,
-  backtext,
-  modaltext,
-  modbutton,
-  backbutton,
+  BackText,
+  OpenModalText,
+  ModalText,
 } from './styles';
+import {useDispatch, useSelector} from 'react-redux';
+import {AppDispatch, RootState} from '../../../store';
+import {GET_CRYPTOS} from '../../../store/reducer/RootReducer';
+import LocalCryptos from '../../../store/localAPI/local';
+import {cryptoType} from '../../../store/types/CryptoTypes';
+import {theme} from '../../utils/theme';
 
 const AddCrypto = () => {
   const [text, onChangeText] = useState('');
   const [modalVisible, setModalVisible] = useState(false);
+  const [borderColor, setBorderColor] = useState(theme.colors.gray);
```

Comment:
All the validations in this file, are redundant, bad implemented or unnecessaries. please redo.

Remenber you can use ternaries, use const, and if you have a return in a if is not neededto put elses

Reply:
and remember bind